packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts

```diff
@@ -11,8 +11,8 @@ export default ({ api }: { api: APIType }) => {
       const data = yield call(api.getDCProducts)
       yield put(A.getProductsSuccess(data))
     } catch (e) {
-      const error = errorHandler(e)
-      yield put(A.getProductsFailure(error))
+      console.error('Failed to get card products', errorHandler(e))
```

Comment:
any reason to keep console here?

Reply:
I didn't want to miss this information and, in this flow, there is no message shown to the user. However, it might be wrong to do this. I couldn't find a clear idea in the project on how this things should be managed

Reply:
After DM with Milan decided to leave it